packages/rml/rml.1.09.07/opam

```diff
@@ -27,9 +27,9 @@ In ReactiveML, the reactive model is integrated at the language level
 (not as a library) which leads to safer and more natural programming."""
 authors: "Louis Mandel louis@reactiveml.org"
 url {
-  src: "http://rml.lri.fr/distrib/rml-1.09.07-2021-07-26.tar.gz"
+  src: "https://github.com/reactiveml/rml/archive/refs/tags/rml-1.09.07-2021-07-26.tar.gz"
   checksum: [
-    "sha256=74acc5319df33b4694d6894b0b8023c1ce3f8f0749c6f4dc587a42c102d809b0"
-    "md5=4fc5ca8f941f6a7f060fb9252389b8b4"
+    "sha256=5beeb70dbd8688597999377b54342c1306ac92a3eefb68978a38311c531de5ca"
```

Comment:
If the hashes have changed, this should also be marked as unavailable, unless we can find one in the cache to compare the contents

Reply:
Ok, I undid the changes and just marked it as unavailable.